src/chocolatey/infrastructure.app/services/NugetService.cs

```diff
@@ -1014,6 +1020,8 @@ public virtual void EnsureBackupDirectoryRemoved(string packageName)
 
                     string logMessage = @"{0} is not installed. Installing...".FormatWith(packageName);
 
+                    localPackageListValid = false;

```

Comment:
Is there any particular reason for this to have the empty line after the logMessage?

Reply:
No specific reason.